ast-common/src/org/jetbrains/dukat/astCommon/NameEntity.kt

```diff
@@ -4,12 +4,20 @@ sealed class NameEntity : Entity
 
 data class IdentifierEntity(
         val value: String
-) : NameEntity()
+) : NameEntity() {
+    override fun toString(): String {
```

Comment:
I'm against of overriding toString.
However! Checkout NameEntity.translate family of methods - I can imagine that we'll move it somewhere nearer to NameEntity.